it-tools.yaml

```diff
@@ -74,8 +74,31 @@ update:
     tag-filter: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - curl
+        - shadow
+        - wait-for-it
   pipeline:
     - name: Verify nginx
       runs: |
         stat /etc/nginx/conf.d/default.conf
         stat /usr/share/nginx/html/index.html
+    # Using manual runs instead of test/daemon-check-output since nginx doesn't emit usable stdout on startup.
+    - name: "Start and verify nginx manually"
+      runs: |
+        useradd -r -s /sbin/nologin nginx 2>/dev/null
+        mkdir -p /var/lib/nginx/tmp/client_body
+        mkdir -p /var/log/nginx
+        mkdir -p /var/run
+
+        nginx -g 'daemon off;' > /tmp/nginx.log 2>&1 &
```

Comment:
Let's migrate to `test/daemon-check-output` pipeline.

Reply:
@Dentrax! initially i made use of `test/daemon-check-output`, but nginx doesn't emit any consistent output to stdout/stderr on successful startup. This caused the `expected_output` check to fail. Additionally to ensure compatibility, especially with QEMU runners, I opted for a manual background start .